jwst/cube_build/ifu_cube.py

```diff
@@ -538,14 +545,23 @@ def build_ifucube(self):
                 if self.interpolation == 'pointcloud':
                     t0 = time.time()
                     pixelresult = self.map_detector_to_outputframe(this_par1,
-                                                                   this_par2,
                                                                    subtract_background,
                                                                    ifile)
 
-                    coord1, coord2, wave, flux, rois_pixel, roiw_pixel, weight_pixel,\
+                    coord1, coord2, wave, flux, slice, rois_pixel, roiw_pixel, weight_pixel,\
                         softrad_pixel, alpha_det, beta_det = pixelresult
                     t1 = time.time()
                     log.info("Time to transform pixels to output frame = %.1f.s" % (t1 - t0,))
+
+# If setting the DQ plane of the IFU
+                    if self.skip_dqflag:
+                        log.info("Skipping setting DQ flagging")
+                    else:
+                        t0 = time.time()
+                        roiw_ave = np.mean(roiw_pixel)
+                        self.map_fov_to_dqplane(this_par1, coord1, coord2, wave, roiw_ave, slice)
+                        t1 = time.time()
+                        log.info("Time to set initial dq values = %.1f.s" % (t1 - t0,))
```

Comment:
This log message could perhaps be downgraded to "debug" status. It's not really necessary for all users to see it every time it's run.